layers/+irc/erc/packages.el

```diff
@@ -214,7 +214,8 @@
   (use-package erc-image
     :init (with-eval-after-load 'erc (add-to-list 'erc-modules 'image))))
 
-(defun erc/init-erc-terminal-notifier ())
+(defun erc/init-erc-terminal-notifier ()
+  (use-package erc-terminal-notifier))
```

Comment:
Looks fine in erc layer. Although you may benefit by adding `:if (executable-find "terminal-notifier")` to use-package to selectively enable only if the executable is found.